sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQueryFileTest.scala

```diff
@@ -45,25 +45,34 @@ abstract class HiveQueryFileTest extends HiveComparisonTest {
     runOnlyDirectories.nonEmpty ||
     skipDirectories.nonEmpty
 
-  val whiteListProperty: String = "spark.hive.whitelist"
-  // Allow the whiteList to be overridden by a system property
-  val realWhiteList: Seq[String] =
-    Option(System.getProperty(whiteListProperty)).map(_.split(",").toSeq).getOrElse(whiteList)
+  val deprecatedIncludeListProperty: String = "spark.hive.whitelist"
+  val includeListProperty: String = "spark.hive.includelist"
```

Comment:
did you see what was using this? I don't see it used anywhere internally.

Reply:
I believe it would be a user-provided string. I was going back and forth on whether or not maintaining it as a deprecated property is necessary. From my reading, this is a developer-only property that would be applied during a testing/development cycle. I tried to be as cautious as possible to maintain compatibility to any contention, but in this case it seems it may be unnecessary. I would love to hear your thoughts.

Reply:
I think leaving it as deprecated for at least one release cycle is good, makes upgrading easier.

Reply:
I don't know how its being used probably safest to leave it in this context.  Ideally if its testing config would rather see it call something like spark.test.X and documented on the usage of it, but I think your change is fine for now.

Reply:
OK, will leave the deprecation for now. Thanks for the discussion!